charts/ExploreView.tsx

```diff
@@ -26,14 +19,24 @@ import { Indicator } from "./Indicator"
 import { RootStore, StoreEntry } from "./Store"
 
 function chartConfigFromIndicator(
-    indicator: Partial<Indicator>
+    indicator: Indicator
 ): Partial<ChartConfigProps> {
-    return indicator
+    return {
+        ...indicator,
+        // TODO need to derive selected data from ExploreModel, since selections
+        // should persist when switching indicators.
+        selectedData: [
+            {
+                index: 0,
+                entityId: 355
+            }
+        ]
+    }
 }
 
-function noop(value: any): void {
-    return
-}
+import * as urlBinding from "charts/UrlBinding"
+import { ExploreUrl } from "./ExploreUrl"
+import { ExploreModel, ExplorerChartType } from "./ExploreModel"
```

Comment:
All imports should be at the top, no?

Reply:
Ah, didn't notice when merging. Now fixed, thanks!